docusaurus/src/components/HeaderDecoration.jsx

```diff
@@ -91,11 +154,28 @@ export const HeaderDecoration = ({
   originalTitle,
   originalId,
   github_url,
+  cdkVersion,
+  isLatestCDKString,
+  cdkVersionUrl,
+  syncSuccessRate,
+  usageRate,
+  lastUpdated,
 }) => {
   const isOss = isOssString.toUpperCase() === "TRUE";
   const isCloud = isCloudString.toUpperCase() === "TRUE";
   const isPypiPublished = isPypiPublishedString.toUpperCase() === "TRUE";
   const isArchived = supportLevel.toUpperCase() === "ARCHIVED";
+  const isLatestCDK = isLatestCDKString.toUpperCase() === "TRUE";
```

Comment:
> Separate question: why are isOss, isCloud, isPypiPublished, and isLatestCDKString string values instead of boolean values? Seems like booleans would make more sense here.

So! This was a weird system implementation detail I had to discover.

The way remark passes these values to the <HeaderDecoration /> is through raw html attributes it adds to an element during generation.

This leads to some odd behaviour where
1. boolean values aren't translate properly
2. undefineds turn into true

What I did to address is 
1. filter out undefined values before appending to the html element
2. created two helper functions for translating between boolean strings

> Looking [here](https://github.com/airbytehq/airbyte/pull/38799/files#diff-c8c2d902770fab687d48ecc1481bb45950ff556cdb56a0cc63f52bc05ba10ac2R42-R44) it seems those string values are coming from the registy, but I think we should convert them to boolean values in that connector_registry.js file so that this component can receive nice boolean values

This is a read detail, as the value is correct in the report. 
![image](https://github.com/airbytehq/airbyte/assets/1325608/4b9853d5-793d-4362-bce0-1b84af6f4d94)
So it seems like `true` in the registry report becomes `"true"` when we process it.
Likely a bug in how we read the file.

But Im not confident I can remediate that without accidentally introducing another bug in our docs as it seems like we have other code that relies on it being a Boolean String.

Reply:
I see. Maybe if we didn't use remark for this we could avoid that problem, but I don't think its worth refactoring it just for that. Having a comment is fine :+1: